Triton_Inference_Server_Python_API/README.md

```diff
@@ -0,0 +1,265 @@
+r23.12 release of Triton.
+
+> [!Note]
+> As the API is in BETA please expect some changes as we
+> test out different features and get feedback.
+> All feedback is weclome and we look forward to hearing from you!
+
+## Requirements
+
+The following instructions require a linux system with Docker
+installed. For CUDA support, make sure your CUDA driver meets the
+requirements in "NVIDIA Driver" section of Deep Learning Framework
+support matrix:
+https://docs.nvidia.com/deeplearning/frameworks/support-matrix/index.html
+
+## Build / Installation
+
+The tutorial and Python API package are designed to be installed and
+run within the `nvcr.io/nvidia/tritonserver:23.12-py3` docker image.
+
+A set of convenience scripts are provided to create a docker image
+based on the `nvcr.io/nvidia/tritonserver:23.12-py3` image with the
+Python API installed plus additional dependencies required for the
+examples.
+
+### Trition Inference Server 23.12 + Python API
+
+#### Build Image
+```bash
+   ./build.sh
+```
+
+#### Example Output
+```bash
+#18 naming to docker.io/library/triton-python-api:r23.12 0.0s done
+#18 DONE 0.2s
++ [[ TEST == TRT_LLM ]]
++ [[ TEST == TEST ]]
++ mkdir -p /home/user/tutorials/Triton_Inference_Server_Python_API/models
++ cp -rf /home/user/tutorials/Triton_Inference_Server_Python_API/deps/test/test_api_models/test /home/user/tutorials/Triton_Inference_Server_Python_API/models/.
+```
+
+#### Supported Backends
+```
+dali  fil  identity  onnxruntime  openvino  python  pytorch  repeat  square  tensorflow  tensorrt
+```
+
+#### Included Models
+
+The default build includes a `test` model that can be used for
+exercising basic operations including sending input tensors of
+different data types. The `test` model copies provided inputs of
+`shape [-1, -1]` to outputs of shape `[-1, -1]`. Inputs are named
+`data_type_input` and outputs are named `data_type_output`
+(e.g. `string_input`, `string_output`).
+
+
+## Hello World
+
+### Start Container
+
+The following command starts a container and volume mounts the current
+directory as `workspace`.
+
+```bash
+   ./run.sh
+```
+
+#### Example Output
+
+```bash
+=============================
+== Triton Inference Server ==
+=============================
+
+NVIDIA Release 23.12 (build 77457706)
+Triton Server Version 2.41.0
+
+Copyright (c) 2018-2023, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+
+Various files include modifications (c) NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+
+This container image and its contents are governed by the NVIDIA Deep Learning Container License.
+By pulling and using the container, you accept the terms and conditions of this license:
+https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license
+
+NOTE: CUDA Forward Compatibility mode ENABLED.
+  Using CUDA 12.3 driver version 545.23.08 with kernel driver version 525.85.12.
+  See https://docs.nvidia.com/deploy/cuda-compatibility/ for details.
+
+root@user-machine:/workspace#
+```
```

Comment:
Instead of this section, we could perhaps say now you should be inside a container with TritonServer In-Process Python package installed and model repository setup for the requirements of this tutorial.